styles/style.css

```diff
@@ -27,18 +27,23 @@ body {
     --footer-height: 30px;
 
     font-family: 'Titillium Web', sans-serif;
+    font-weight:bolder;
     background-color: var(--color2);
 }
 
 main {
+    
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
     margin-bottom: calc(var(--footer-height));
+    display: box;
```

Comment:
Ahhh, you have two display options set here. If you're wanting to use `grid` instead of `flex`, you'll need to set `display: grid` and you should always only have one `display` property set for an element in CSS.